experiments/008/views/match_view.js

```diff
@@ -0,0 +1,68 @@
+class MatchView {
+    constructor(controller) {
+        this.controller = controller;
+        this.match = controller.match;
+        this.el = $('#match-view');
+        this.scoreboardEl = $('table#scoreboard');
+        this.gameboardEl = $('div#gameboard');
+    }
```

Comment:
jQuery does give you a way to "componentize" this view. With template literals, you can get a little crazy: https://stackoverflow.com/a/805113